pkg/acs/calacs/acscte/dopcte-gen2.c

```diff
@@ -93,6 +93,10 @@ int doPCTEGen2 (ACSInfo *acs, CTEParamsFast * ctePars, SingleGroup * chipImage)
         amploc = strchr(AMPSORDER, ccdamp[nthAmp]);
         ampID = *amploc - AMPSORDER[0];
 
+        ctePars->rn_amp = acs->readnoise[ampID];
```

Comment:
@mdlpstsci This single line is the actual fix. Everything else is to just reporting etc.